packages/isar/lib/src/isar_collection.dart

```diff
@@ -39,28 +39,28 @@ abstract class IsarCollection<OBJ> {
   Future<int> put(
     OBJ object, {
     bool replaceOnConflict = false,
-    bool saveLinks = false,
+    bool saveLinks = true,
```

Comment:
I like we are saving link by default now, for some users it takes too much time to figure out that they needed to set this to true.